scripts/cibuild

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${CI}" ]]; then
+    set -x
+fi
+
+function usage() {
+    echo -n \
+"Usage: $(basename "$0")
+
+Build container images from templates.
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+        sed -e "s/%%GDAL_VERSION%%/$GDAL_VERSION/" \
+            -e "s/%%OPENJPEG_VERSION%%/$OPENJPEG_VERSION/" \
+            -e "s/%%ANT_VERSION%%/$ANT_VERSION/" \
+            "Dockerfile-$VARIANT.template" > Dockerfile
+
+        docker build -t "quay.io/azavea/openjdk-gdal:$GDAL_VERSION-$VARIANT" .
```

Comment:
How about:

```
quay.io/azavea/openjdk-gdal:2.3.2-jdk-8
```

So that we can leverage the implied GDAL versioning in `gdal:2.3.2`? That seems to be the pattern we follow with our Django images, anyway: https://quay.io/repository/azavea/django?tag=latest&tab=tags

Reply:
Heh, I went back-and-forth on it and settled on putting the JDK version first because it is first in the container image name. I also used `8-jdk` to align with the official container image naming structure.

This might be a thing where you propose both options to 3 people, see what makes sense to the majority, and just go with that.